flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointRequestDeciderTest.java

```diff
@@ -284,7 +284,7 @@ private CheckpointRequestDecider decider(
                 maxQueued);
     }
 
-    private static final LongConsumer NO_OP = unused -> {};
+    private static final BiConsumer<Long, Long> NO_OP = (unused, unused2) -> {};
```

Comment:
It seems hack to name variables as unused2. How about using this:

`(currentTimeMillis, tillNextMillis) -> {}`